python/foglamp/tasks/purge/purge.py

```diff
@@ -105,17 +104,22 @@ def __init__(self):
         start_time = time.strftime('%Y-%m-%d %H:%M:%S.%s', time.localtime(time.time()))
 
         payload = PayloadBuilder().AGGREGATE(["count", "*"]).payload()
-        result = await self._storage_async.query_tbl_with_payload("readings", payload)
+        result = await self._readings_storage_async.query(payload)
```

Comment:
This is good, the previous code was wrong as if we had a separate readings plugin it woudl not have worked.